services/ansibler/server/server.go

```diff
@@ -78,12 +78,14 @@ func (*server) TeardownLoadBalancers(ctx context.Context, req *pb.TeardownLBRequ
 		attached        = make(map[string]bool)           // [k8sClusterName]Bool
 		k8sNodepools    = make(map[string][]*pb.NodePool) //[k8sClusterName][]nodepools
 		k8sNodepoolsKey = make(map[string]string)         //[k8sClusterName]keys
+		clusterIDs      = make(map[string]string)         //[k8sClusterName]prefix (clusterName + hash)
```

Comment:
Just for the consistency
`//[k8sClusterName]prefix (clusterName + hash)`  -> `//[k8sClusterName]clusterID (<clusterName>-<hash>)`